pyaedt/edb_core/edb_data/simulation_setup.py

```diff
@@ -0,0 +1,701 @@
+from pyaedt.edb_core.general import convert_pydict_to_netdict
+from pyaedt.generic.general_methods import generate_unique_name
+from pyaedt.generic.general_methods import is_linux
+from pyaedt.generic.general_methods import pyaedt_function_handler
+
+
+def _parse_value(v):
+    """
+
+    Parameters
+    ----------
+    v :
+
+
+    Returns
+    -------
+
+    """
+    #  duck typing parse of the value 'v'
+    if v is None or v == "":
+        pv = v
+    elif v == "true":
+        pv = True
+    elif v == "false":
+        pv = False
+    else:
+        try:
+            pv = int(v)
+        except ValueError:
+            try:
+                pv = float(v)
+            except ValueError:
+                if isinstance(v, str) and v[0] == v[-1] == "'":
+                    pv = v[1:-1]
+                else:
+                    pv = v
+    return pv
+
+
+class BaseSimulationSetup(object):
+    """Ansys.Ansoft.SimSetupData.Data.BaseSimulationSettings."""
```

Comment:
Description for a class should be something like "Provides base simulation settings." The brief description should be in plain English.